apps/views.py

```diff
@@ -89,13 +89,20 @@ def post(self, request, user, project):
                 }
                 res = requests.get(url, params=query)
                 res_json = res.json()["data"]["result"]
-
+                # TODO: change timestamp logic. Timestamps are different in prod and dev
+                if settings.DEBUG is True:
+                    prod_timestamp = False
+                else:
+                    prod_timestamp = True
```

Comment:
Seems a bit unnecessary to create a boolean from a boolean, then use the boolean to do boolean stuff :) I suggest remove this, and commit my suggestion from below

Reply:
Yeah makes sense. I had it in my mind to change this in the morning, got too tired last night.

Reply:
Understandable :)